src/parsl-rp/parsl-rp.ipynb

```diff
@@ -173,10 +163,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# we ask Parsl to start the executor locally with 4 cores\n",
-    "config = Config(\n",
-    "    executors=[RadicalPilotExecutor(resource='local.localhost_test', access_schema='local',\n",
-    "                                    project='', partition='', walltime=30, cores=4, bulk_mode=True)])\n",
+    "config = Config(executors=[RadicalPilotExecutor(\n",
```

Comment:
there is no `ResourceConfig` here (but the text above talks about it) + review that `to allocate 4 cores for all tasks` in a similar way as I mentioned in the comment above